doc/factory/custom_vars.html

```diff
@@ -1701,6 +1701,32 @@ <h3><a name="frontend_vars"></a>Frontend Client Variables</h3>
               </p>
             </td>
           </tr>
+          <tr>
+            <td><b>GLIDEIN_BLACKHOLE_NUMJOBS</b></td>
+            <td>Int</td>
+            <td>
+              <p>
+                This attribute represents number of jobs completed too quickly
+                before raising the blackhole detection flag. More details in
+                blackhole detection section, on the bottom of who webpage
+                (Lifetime of a Glidein).
+              </p>
+            </td>
+          </tr>
+          <tr>
+            <td><b>GLIDEIN_BLACKHOLE_RATE</b></td>
+            <td>Int</td>
```

Comment:
Should we say that this is a float? String for the shell, used as floating point number in the condor config. This would allow for slower rates (less than one per second)

Reply:
updated as suggested